raresGabriel01/home-assignment
# Issue 7

# Description
Added a pre-commit hook that automatically formats the code when commiting.

# Issue link(s)
#7 

# How to test?

1. Write some messy code
2. Checkout on a new branch 
3. run `git add . && git commit -m "some_message"` in the root folder
4. Check to see if your messy code got formatted correctly

# Attachments 
N/A
